content/en/docs/setup/install/istioctl/index.md

```diff
@@ -270,16 +270,19 @@ $ istioctl manifest apply -f samples/operator/pilot-k8s.yaml
 {{< /text >}}
 
 {{< tip >}}
-For backwards compatibility, the previous [Helm installation options](/docs/reference/config/installation-options/)
-are also fully supported. To set them on the command line, prepend the option name with "`values.`".
+For backwards compatibility, most of the previous [Helm installation options](/docs/reference/config/installation-options/)
+are also supported except for kubernetes resources. To set them on the command line, prepend the option name with "`values.`".
```

Comment:
This sentence is now ambiguous. "To set them ..." could be interpreted to be either the helm installation options or the kubernetes resources. How about:
```
For backwards compatibility, the previous [Helm installation options](/docs/reference/config/installation-options/),
with the exception of Kubernetes resource settings,
are also fully supported. To set them on the command line, prepend the option name with "`values.`".
```

Reply:
ok will do